packages/cli/src/commands/remove.ts

```diff
@@ -167,9 +169,15 @@ export default async function main(client: Client) {
         })
       );
 
-      projectDeployments
-        .slice(0, 201)
-        .map(pDeployments => deployments.push(...pDeployments));
+      const to = Math.min(projectDeployments.length, 201);
+      for (let i = 0; i < to; i++) {
+        for (const pDepl of projectDeployments[i]) {
+          deployments.push({
+            ...pDepl,
```

Comment:
Can we avoid spread here and assign the prop instead?

Reply:
Fixed